### DISCUSSION TITLE
Lammps is simulated with too much memory

### DISCUSSION THREAD

**Discussion by manbujin** [deepmodeling/dpgen#1525] [Q&A] [answered]
Hello！
I trained the potential function of nine elements, and when I did a lammps simulation of 1,700,000 atoms, it took up almost 1T of memory. I've compressed the dp model, but it still takes up a lot of memory, is there any solution for this?
Thank you very much for your answer.

**Comment by njzjz** [comment, marked as answer]
You can consider setting the precision to `float32` to save memory. xref: https://github.com/deepmodeling/deepmd-kit/discussions/3805

**Reply by manbujin** [reply]
Thank you very much, it effectively reduces the memory footprint.
